deploy/crds/postgresql_v1alpha1_database_crd.yaml

```diff
@@ -2,6 +2,18 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: databases.postgresql.baiju.dev
+  annotations:
+    servicebindingoperator.redhat.io/spec.dbName: 'binding:env:attribute'
```

Comment:
@pmacik in general, I would not add any quotes to a string in YAML. Just a minor comment in this PR :-)

Reply:
@otaviof Thx for the tip... why not? Does it affect anything in a bad way? :)

Reply:
@pmacik nope, it's just cosmetics :-)